### ISSUE TITLE
Skipped files (using %md5% tag), and cant log in

### ISSUE THREAD

**Issue by GreenAlpha01** [Nandaka/DanbooruDownloader#233]
I'm getting a bunch of skipped files while downloading from danbooru and was wondering if this is because there are simply some file formats the downloader isn't compatible with?

edit: I also realized im not actually logged in, and this could be why it's skipping files, but for the life of me I cant figure out how to log in for danbooru.

**Comment by Nandaka** [comment]
got the sample md5? fyi: you need to copy the cookie value similar like sankaku for danbooru from now. I think they kill the old login api.

**Comment by GreenAlpha01** [comment]
> fyi: you need to copy the cookie value

Yeah I've tried that but i can't find any cookie value, i might just be dumb tho.
